core/src/stages/connect.cpp

```diff
@@ -219,9 +219,16 @@ SubTrajectoryPtr Connect::merge(const std::vector<robot_trajectory::RobotTraject
                                 const std::vector<planning_scene::PlanningSceneConstPtr>& intermediate_scenes,
                                 const moveit::core::RobotState& state)
 {
+	// set cost
+	double cost = 0;
+	for (const auto& trajectory: sub_trajectories) {
+		for (const double& distance: trajectory->getWayPointDurations())
```

Comment:
```suggestion
		cost += trajectory->getWayPointDurationFromStart();
```

Reply:
There is no overload for this without a parameter and `trajectory->getWayPointDurationFromStart(trajectory->getWayPointCount())`
is not any more readable or efficient than the current one.

A new `trajectory->getDuration()` function in MoveIt would be appreciated, as the class already provides a `getAverageSegmentDuration()` too...